src/Table/index.jsx

```diff
@@ -63,24 +63,28 @@ class Table extends React.Component {
   }
 
   getTableHeading(column) {
-    return (
-      this.props.tableSortable && column.columnSortable ?
-        <Button
-          label={
-            <span>
-              {column.label}
-              <span className={classNames(styles['sr-only'])}>
-                {' '}
-                {this.getSortButtonScreenReaderText(column.key)}
-              </span>
+    let heading;
+    if (this.props.tableSortable && column.columnSortable) {
+      heading = (<Button
+        label={
+          <span>
+            {column.label}
+            <span className={classNames(styles['sr-only'])}>
               {' '}
-              {this.getSortIcon(column.key === this.state.sortedColumn ? this.state.sortDirection : '')}
-            </span>}
-          onClick={() => this.onSortClick(column.key)}
-        />
-        :
-        column.label
-    );
+              {this.getSortButtonScreenReaderText(column.key)}
+            </span>
+            {' '}
+            {this.getSortIcon(column.key === this.state.sortedColumn ? this.state.sortDirection : '')}
+          </span>}
+        onClick={() => this.onSortClick(column.key)}
+      />);
+    } else if (column.hide) {
```

Comment:
Nitpick: would a better name be `column.hideHeader`? `column.hide` makes it sound like you want to hide the column.

And if we ever want to implement a feature in the future that does actually hide a column, it would be pretty confusing to implement it and still maintain this particular feature.

Reply:
good idea!